### DISCUSSION TITLE
VideoSource is not playing

### DISCUSSION THREAD

**Discussion by MuongKimhong** [pixijs/pixijs#10978] [Q&A]
Hello, I'm trying to play a video with `VideoSource` but for some reasons it's not playing.

```javascript
let videoElement = document.createElement("video");

videoElement.addEventListener("canplay", () => {
    const source = new VideoSource({
      alphaMode: "no-premultiply-alpha",
      updateFP: 30,
      loop: true,
      preload: true,
      autoPlay: true,
      resource: videoElement,
    });

    const texture = new Texture({
      source,
    });

    let sprite = new Sprite(texture);
    sprite.width = window.innerWidth;
    sprite.height = window.innerHeight;
    mycanvas.stage.addChild(sprite);
    console.log(source.resource.playbackRate); // get 1
    console.log(source._msToNextUpdate); //  get 0
    console.log(Ticker.shared.elapsedMS); // get 16.66666
});

videoElement.src = "myvideo.mp4";
```
I managed to display the video on screen but it's not playing. I have gone thru Docs and Codebase, code has no errors logged, and I can't figure out why, so I decided to ask for help. Thanks

**Comment by codeerspl** [comment]
Working example:

          const videoElement = document.createElement('video')
          videoElement.src = assetData.src

          const videoSource = new VideoSource({
            updateFPS: 60,
            muted: false,
            loop: false,
            autoPlay: true,
            crossorigin: true,
            preload: true,
            resource: videoElement
          })

          videoSource.load().then( loadedSource => {
                const videoTexture = new Texture({ source: loadedSource })          
                const videoSprite = new Sprite(videoTexture)
                
                // Add sprite to the stage
          });
